server/apps/distributed-app/src/main/java/org/apache/james/CassandraRabbitMQJamesServerMain.java

```diff
@@ -127,7 +129,7 @@
         new CassandraQuotaMailingModule());
 
     private static final Module BLOB_MODULE = Modules.combine(
-        new BlobStoreAPIModule(),
+        new BlobStrategyModule(),
```

Comment:
Can we try to bing the GC task **only if** deduplication is enabled?

If we do not have deduplication, it is mostly useless...

We can do that by moving the bindings somewhere within the BlobStore module chooser, in the modules responsible for deduplication...